is_prime.py

```diff
@@ -0,0 +1,9 @@
+def is_prime(n):
```

Comment:
Optional, but I would try to take a look at getting into a habit of using type hints for your function signatures. Can use these for reference, if you are not familiar

https://docs.python.org/3/library/typing.html

https://realpython.com/lessons/type-hinting/